README.md

```diff
@@ -61,3 +61,109 @@ internxt
 - If you encounter any issues, verify that your environment matches the prerequisites above.
 
 With these steps, your Windows application setup will be complete and ready to use.
+
+---
+
+# How to Manually Sign and Effectively Publish a Release
+
+**IMPORTANT: READ THE ENTIRE PROCESS CAREFULLY BEFORE PROCEEDING.**
+
+If you plan to manually sign a build for publication, follow these steps:
+
+## Step 1: Generate the Build
+
+Run the command:
+
+```bash
+yarn package
+```
+
+This will generate an unsigned build and the following files:
+
+- `Internxt Drive Setup 2.3.5.exe`
+- `Internxt Drive Setup 2.3.5.exe.blockmap`
+- `latest.yml`
+
+## Step 2: Sign the `.exe` File
+
+You need to sign the `.exe` file using either DigiCert tools or the `smctl` CLI.
+
+### Requirements:
+
+- **The `.p12` certificate**
+- **The `.p12` password**
+- **The API key**
+
+Follow the setup instructions in this guide:
+[smctl Setup Guide](https://docs.digicert.com/en/digicert-keylocker/client-tools/signing-tools/smctl.html).
+
+### Key Steps:
+
+1. Download and install **smctl** or **DigiCert​​®​​ KeyLocker**.
+
+   - The MSI file can be requested from Fran, Sergio, or Jonathan.
+
+2. If the DigiCert application does not open, navigate to:
+   `C:\Program Files\DigiCert\Click to Sign`
+
+3. Open the **Click to Sign** application to configure the required environment variables.
+
+### Signing the Application:
+
+Once configured, right-click on the `.exe` file:
+`Show More Options -> DigiCert@ -> Sign Now`.
+
+This will sign the application.
+
+---
+
+## Step 3: Validate the SHA512 Hash
+
+**IMPORTANT:** Signing the application will modify its SHA512 hash.
+
+To validate the new hash, use the following command:
+
+```bash
+CertUtil -hashfile ".\Internxt Drive Setup 2.3.5.exe" SHA512
+```
+
+This will output the new hash.
+You can compare the hash before and after signing to confirm the modification.
+
+---
+
+## Step 4: Update the `latest.yml` File
+
+The `latest.yml` file contains the SHA512 hash in Base64 format.
```

Comment:
Explain why this step needs to be done according to how electron-updater behaves, so we do not need to guess it